src/internal/internal_hettmqr.cc

```diff
@@ -158,44 +159,57 @@ void hettmqr(
                     C.tileGetForWriting(i, i, LayoutConvert(layout));
                     make_hermitian( C(i, i) );
                     int dst = C.tileRank(i2, i1);
+                    int tag = tag_base + i + i*C.mt();
                     C.tileSend(i, i, dst, tag);
                     C.tileRecv(i, i, dst, layout, tag);
                 }
             }
             if (C.tileIsLocal(i2, i1)) {
-                // Receive tiles from sources, apply Q on both sides,
-                // then send updated tiles back.
-                int src11 = C.tileRank(i1, i1);
-                C.tileRecv(i1, i1, src11, layout, tag);
-
-                int src22 = C.tileRank(i2, i2);
-                C.tileRecv(i2, i2, src22, layout, tag);
-
-                V.tileGetForReading(i2, 0,  LayoutConvert(layout));
-                T.tileGetForReading(i2, 0,  LayoutConvert(layout));
-                C.tileGetForWriting(i2, i1, LayoutConvert(layout));
-
-                // Workspace C(i1, i2) = C(i2, i1)^H.
-                C.tileInsert(i1, i2);
-                tile::deepConjTranspose( C(i2, i1), C(i1, i2) );
-
-                int64_t nb = std::min(V.tileMb(i2), V.tileNb(0));
-                tpmqrt(Side::Left, op, nb,
-                       V(i2, 0), T(i2, 0), C(i1, i1), C(i2, i1));  // 1st col
-                tpmqrt(Side::Left, op, nb,
-                       V(i2, 0), T(i2, 0), C(i1, i2), C(i2, i2));  // 2nd col
-                tpmqrt(Side::Right, opR, nb,
-                       V(i2, 0), T(i2, 0), C(i1, i1), C(i1, i2));  // 1st row
-                tpmqrt(Side::Right, opR, nb,
-                       V(i2, 0), T(i2, 0), C(i2, i1), C(i2, i2));  // 2nd row
-
-                C.tileSend(i1, i1, src11, tag);
-                C.tileSend(i2, i2, src22, tag);
-                if (call_tile_tick) {
-                    C.tileTick(i1, i1);
-                    C.tileTick(i2, i2);
+                #pragma omp task
```

Comment:
Maybe picky but no additional info for the tasks here like `slate_omp_default_none`. Is it okay? I did not check.